tests/test_globalization.py

```diff
@@ -106,47 +111,107 @@ def run_charset_validation(self, characters_list):
 
 class GlobalizationMetadataTests(unittest.TestCase):
 
-    def test_schema_metadata_double(self):
-        charset = GB18030()
-        characters_list = charset.get_next_characters(
-            charset.get_double_characters())
-        self.run_schema_metadata_validation(characters_list)
-
-
-    def run_schema_metadata_validation(self, characters_list):
-        client = create_mssql_cli_client()
-        completer = MssqlCompleter(smart_completion=True)
-        new_schemas = []
+    def test_schema_metadata(self):
+        characters_list = get_GB18030_characters_list()
         try:
+            client1 = create_mssql_cli_client()
```

Comment:
Can you move the create database operation into the mssqltestutils.py so that database creation is done in one place? Ideally, it's returns a object with a Drop method and Name property, but we can start with just returning the name.

Reply:
fixed as requested.